stdlib/3/multiprocessing/dummy/__init__.pyi

```diff
@@ -19,7 +19,14 @@ class DummyProcess(threading.Thread):
     _pid: None
     _start_called: int
     exitcode: Optional[int]
-    def __init__(self, group=..., target=..., name=..., args=..., kwargs=...) -> None: ...
+    def __init__(
+        self,
+        group: None = ...,
```

Comment:
I think it's better to use `Any` or whatever other process classes accept here. The argument is ignored for `DummyProcess`, but I surmise it exist for compatibility with other process classes.

Reply:
I made the recommended change.